quill-cassandra/src/main/scala/io/getquill/sources/cassandra/cluster/ClusterBuilder.scala

```diff
@@ -58,10 +59,14 @@ object ClusterBuilder {
     instance
   }
 
+  val stringArrayClass = java.lang.reflect.Array.newInstance(classOf[String], 0).getClass()
+
   private def param(key: String, tpe: Class[_], cfg: Config) =
     Try {
       if (tpe == classOf[String])
         cfg.getString(key)
+      else if (tpe == stringArrayClass)
+        cfg.getStringList(key).asScala.toArray
```

Comment:
simply calling `toArray` without explicit `asScala` did not work for some reason